src/handlers/clean/handler.ts

```diff
@@ -34,63 +36,39 @@ export const handler = async (event: SQSEvent): Promise<Response> => {
 
 async function cleanRecord(record: SQSRecord): Promise<void> {
   const bodyObject = JSON.parse(record.body);
-  if (
-    typeof bodyObject?.component_id !== "string" ||
-    typeof bodyObject?.timestamp !== "number"
-  ) {
-    const message = "Invalid record body.";
-    console.error(message);
-    throw new Error(message);
-  }
 
-  if (
-    process.env.OUTPUT_QUEUE_URL === undefined ||
-    process.env.OUTPUT_QUEUE_URL.length === 0
-  ) {
-    const message = "Output queue URL not set.";
-    console.error(message);
-    throw new Error(message);
+  if (typeof bodyObject !== "object")
+    throw new Error("Event record body not an object.");
```

Comment:
do you need this given that `isValidBodyObject` has `typeof x === "object"`?

Reply:
I don't - good spot. Removed now.